rust/bridge/node/futures/tests/node-tests/bench.js

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2021 Signal Messenger, LLC.
+// SPDX-License-Identifier: AGPL-3.0-only
+//
+
+/* eslint-env es2017 */
+
+const Benchmark = require('benchmark');
+
+const Native = require(process.env.SIGNAL_NEON_FUTURES_TEST_LIB);
+
+const suite = new Benchmark.Suite();
+suite
+  .add('await Native.incrementCallbackPromise(async () => 7)', {
+    defer: true,
+    fn: async deferred => {
+      await Native.incrementCallbackPromise(async () => 7);
+      deferred.resolve();
+    },
+  })
+  .on('complete', function() {
+    this.forEach(test => {
+      console.log(
+        '"' + test.name + '": ' + JSON.stringify(test.times, null, 2)
+      );
+    });
+  })
+  .run();
```

Comment:
```suggestion
  .on('cycle', function(event) {
    console.log(String(event.target));
  })
  .run();
```

Looks a bit nicer:

```
await Native.incrementCallbackPromise(async () => 7) x 29,782 ops/sec ±3.62% (74 runs sampled)
```

Reply:
Yeah, much better! Their docs are, uh, not wonderful.

Reply:
Yeah, I was trying to look up what `defer: true` meant and found it in their tests :joy: